src/MunicipalityRegistry.Api.Projector/paket.template

```diff
@@ -0,0 +1,28 @@
+type file
+version 1.0.0
+
+id Be.Vlaanderen.Basisregisters.MunicipalityRegistry.Api.Projector
+title Be.Vlaanderen.Basisregisters.MunicipalityRegistry.Api.Projector
+
+authors Basisregisters Vlaanderen
+owners agentschap Informatie Vlaanderen <https://vlaanderen.be/informatie-vlaanderen>
+copyright Copyright (c) agentschap Informatie Vlaanderen
+requireLicenseAcceptance false
+projectUrl https://github.com/Informatievlaanderen/municipality-registry
+iconUrl https://raw.githubusercontent.com/Informatievlaanderen/build-pipeline/master/logo.png
+licenseUrl https://opensource.org/licenses/EUPL-1.2
+
+description Municipality Registry Projector API.
+
+dependencies
+  framework: netstandard20
+    Be.Vlaanderen.Basisregisters.Api >= LOCKEDVERSION
```

Comment:
For consistency we agreed to use `~>` for all `Be.Vlaanderen` references with their version as written in `paket.dependencies` instead of `LOCKEDVERSION`

Reply:
fixed for all paket.template files